.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+<insert format ref here>
```

Comment:
Oh, I see it is there, but it is empty now. 👍

Reply:
Actually, I think it needs to be added in a separate pr, because this one will get rebased and invalidate that ref.